extmod/nimble/modbluetooth_nimble.c

```diff
@@ -844,6 +844,7 @@ STATIC int ble_gatt_attr_read_cb(uint16_t conn_handle, const struct ble_gatt_err
     if (error->status == 0) {
         gattc_on_data_available(MP_BLUETOOTH_IRQ_GATTC_READ_RESULT, conn_handle, attr->handle, attr->om);
     }
+    mp_bluetooth_gattc_on_read_write_status(MP_BLUETOOTH_IRQ_GATTC_READ_STATUS, conn_handle, attr->handle, error->status);
```

Comment:
(just double-checking)
This posts a `READ_STATUS` for successful reads, too. Is this also the behavior of btstack?
Is the order of events (result first, then status) the same on btstack?

Reply:
Yeah, they come as separate events on btstack, and you only get the status on the "complete" event, so this was the easiest way to make it consistent between the two.